.goreleaser.yml

```diff
@@ -22,6 +22,12 @@ builds:
   ignore:
     - goos: darwin
       goarch: '386'
+    - goos: freebsd
```

Comment:
Hmm, I don't see those in https://github.com/hashicorp/terraform-provider-scaffolding/blob/6908b6b7ada594861f7db42f3d47ad37a221e416/.goreleaser.yml and this file is directly pulled from there.

Is this something we could report/contribute? Why is this needed in general?

Reply:
I agree, I looked at the same reference project in the last couple of days and noticed that it only excludes macOS i386 flavour.

To be honest, I don't know at which stage the filtering happens, but I can only see a subset of flavours in terraform registry. I checked terraform itself and AWS provider. 

The only reason I made this change in the config is because goreleaser CI task takes ages and I wanted to speed it up a bit.

Reply:
Make sense. It seems here are recommended builds documented: https://www.terraform.io/registry/providers/os-arch

That gives the following matrix:
![Selection_529](https://user-images.githubusercontent.com/16539896/151961768-ab8c3508-592c-4d77-8cc2-2e91f1b4b77c.png)

I'm fine with updating the config according to the matrix.

Do you mind creating an issue about it in https://github.com/hashicorp/terraform-provider-scaffolding? I guess it would be nice if the documentation and configs could be aligned.

Reply:
Oh, great find. I haven't come across that page before. I'll raise a PR upstream in the scaffolding project. I did suspect that Windows ARM flavours were not needed either.

Reply:
I wonder if we should also consider platforms mentioned https://releases.hashicorp.com/terraform/1.1.4/. All that doesn't seem very consistent.

Reply:
That is exactly what I based my original exclusions on. I looked at the same page yesterday.

Reply:
Let's leave FreeBSD arm build then, others can be ignored :+1: And please put links to https://releases.hashicorp.com/terraform/1.1.4/ and https://www.terraform.io/registry/providers/os-arch in a comment, so we know from where this all comes from.

Reply:
OK, to summarise:

1. I shall keep the goreleaser config as it is on the PR now
2. I shall add clarifying comments to the list of excluded flavours
3. I shall raise an issue or PR upstream in the scaffolding project to update the reference implementation

I'll do 2. a bit later in the afternoon (UK time zone).

Reply:
We can also ignore darwin 386 I think? Otherwise sounds good.

Reply:
Done, sorry that it took more than a day to add a couple of lines :)
I have raised https://github.com/hashicorp/terraform-provider-scaffolding/issues/91 upstream as well.